examples/light-switch-app/light-switch-common/light-switch-app.matter

```diff
@@ -3107,7 +3107,7 @@ endpoint 0 {
 
   server cluster DiagnosticLogs {
     ram      attribute featureMap default = 0;
-    ram      attribute clusterRevision default = 1;
+    callback attribute clusterRevision;
```

Comment:
featureMap should also be callback I believe - we have no feature maps to carry over from ember